src/mon/Monitor.cc

```diff
@@ -2652,6 +2664,13 @@ void Monitor::handle_command(MonOpRequestRef op)
   boost::scoped_ptr<Formatter> f(Formatter::create(format));
 
   get_str_vec(prefix, fullcmd);
+
+  // before get fullcmd[0], verify string to vector success.
+  if (fullcmd.empty()) {
```

Comment:
I don't think this check is required. I haven't been able to find a case in which fullcmd can be empty if prefix is non-empty -- and we check whether prefix is empty or not above. if you know of a case, please let me know.